istioctl/cmd/add-to-mesh.go

```diff
@@ -57,6 +57,7 @@ const (
 
 var (
 	crdFactory = createDynamicInterface
+	revision   string
```

Comment:
nit: add a comment here describing which revision this is referencing and why it is important (`revision` and `version` are fairly overloaded at this point).